Userland/Libraries/LibAudio/WavLoader.cpp

```diff
@@ -34,6 +34,8 @@
 
 namespace Audio {
 
+static constexpr size_t maximum_wav_size = 50 * MiB; // FIXME: is there a more appropriate size limit?
```

Comment:
1 minute of 44100Hz 16-bit WAV data is ~10MB so this feels way too short.

Reply:
Whats the current per process memory limit? (and would that be an acceptable size limit)

Reply:
There isn't currently a per-process memory limit, but how about 1GB here? :shrug:

Reply:
done :+1: